nhrpd/nhrp_shortcut.c

```diff
@@ -389,6 +396,16 @@ static void nhrp_shortcut_send_resolution_req(struct nhrp_shortcut *s)
 	    != NHRP_ROUTE_NBMA_NEXTHOP)
 		return;
 
+	if (!retry)
+		s->retry_interval = 1;
+	event_add_timer(master, nhrp_shortcut_retry_resolution_req, s,
+			s->retry_interval, &s->t_retry_resolution);
+	if (s->retry_interval != (NHRPD_DEFAULT_PURGE_TIME / 4))
+		s->retry_interval = ((s->retry_interval * 2) <
```

Comment:
Is this something we should add, or at least a comment on, about what's going on here and why it's calculated this way?

Reply:
Yes, I'll add a comment explaining the interval calculation.